run/config_benchmark.cfg

```diff
@@ -15,5 +15,5 @@ OceanBoundaryModule = Nextsim::BenchmarkOcean
 [ConfigOutput]
 period = P0-0T04:00:00
 start = 2010-01-01T12:00:00Z
-field_names = hsnow,hice,cice,u,v
+field_names = hsnow,hice,cice,u,v,shear
```

Comment:
If adding shear here, it seems like some kind of test of the values should be present in `test_integration_test.py`.

Reply:
This is in the cyclone benchmark config file, so it's not related to the integration test. We should consider making this benchmark an additional integration test, but that's a separate issue.

Reply:
Oh, of course it is.